src/test/java/service/file/CsvFileReaderTest.java

```diff
@@ -0,0 +1,68 @@
+package service.file;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CsvFileReaderTest {
+    private static final String csvFile = "csvFile.csv";
+    private static final String notCsvFile = "notCsvFile.txt";
+    private final FileReader fileReader = new CsvFileReader();
+
+    @BeforeAll
+    static void beforeAll() {
+        String data = "fruit,quantity" + System.lineSeparator()
+                + "b,banana,100" + System.lineSeparator()
+                + "b,apple,123";
+        try {
+            Files.write(Path.of(csvFile), data.getBytes());
+            Files.write(Path.of(notCsvFile), data.getBytes());
+        } catch (IOException e) {
+            throw new RuntimeException("Can`t write some of files");
+        }
+    }
+
+    @Test
+    void validCase_validFileType() {
```

Comment:
Bad naming, [see common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#remember-about-naming-of-test-methods)